app/components/TextWithIcon/index.tsx

```diff
@@ -15,15 +15,16 @@ type Props = {
 
 const TextWithIcon = ({
   iconName,
-  className,
   content,
   tooltipContentIcon,
   iconRight = false,
   size,
 }: Props) => {
   return (
-    <Flex alignItems="center" className={className}>
```

Comment:
The `className` is used other places than on the event page, so the removal of this will break all of those. Any reason for removing?

Reply:
Not really. Added it back now.